### PR TITLE
fix: fix typo in the word interface

### PR DESCRIPTION
**Pull request by EmilIvanov**

### CODE REVIEW

File: assets/express/src/api/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { DbCollection } from 'src/database/intefaces';
+import { DbCollection } from 'src/database/interfaces';
```
Comment (IvayloGrahlyov) [inline assets/express/src/api/index.ts:1]: IMO here we should import with `@database`

File: assets/express/src/api/healthz/routes/live.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { RouteOptions } from 'src/api/intefaces';
+import { RouteOptions } from 'src/api/interfaces';
```
Comment (IvayloGrahlyov) [inline assets/express/src/api/healthz/routes/live.ts:1]: Should we have it like this or use `../../interfaces`, what do you think?

**Review by IvayloGrahlyov — CHANGES_REQUESTED** [review]
Gosh... I'm sorry 😄 IMO we should use the aliases where applicable, though

File: assets/express/src/api/index.ts (hunk 1 above)
Comment (EmilIvanov) [reply assets/express/src/api/index.ts:1]: This PR doesn't aim to fix alias usage

File: assets/express/src/api/healthz/routes/live.ts (hunk 2 above)
Comment (EmilIvanov) [reply assets/express/src/api/healthz/routes/live.ts:1]: This PR doesn't aim for fix imports

**Pull request merged**